src/test/run-pass/traits/trait-alias-syntax.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
I'm not sure I understand why these particular tests were pulled out .. but I don't object I suppose. =)

Reply:
Just because they're testing the various syntactical forms without really testing semantics... I could have left them in, but they were quite independent, so...